build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleJavadocsPlugin.java

```diff
@@ -61,62 +61,42 @@ private void generateJavadocs(Project project, ProjectLayout layout, TaskContain
         // TODO: This breaks if version is changed later
         Object version = project.getVersion();
 
-        TaskProvider<Javadoc> javadocAll = tasks.register("javadocAll", Javadoc.class, task -> {
+        TaskProvider<Copy> javadocAll = tasks.register("javadocAll", Copy.class, task -> {
+            task.from(extension.getJavadocs().getJavadocFontDir().get().getAsFile());
```

Comment:
❓ Can we double check whether the javadoc task supports adding additional resources? I would imagine it's possible, though I don't know for sure. It seems like a smell that we need to introduce a Copy task here.

Reply:
I couldn't find anything when I looked in the code/docs.